File: src/main/java/com/dyonovan/jatm/common/container/machine/ContainerThermalBinder.java
package com.dyonovan.jatm.common.container.machine;

import com.dyonovan.jatm.common.container.BaseContainer;
import com.dyonovan.jatm.common.tileentity.machine.TileThermalBinder;
import net.minecraft.entity.player.InventoryPlayer;
import net.minecraft.inventory.Slot;

public class ContainerThermalBinder extends BaseContainer {

    public ContainerThermalBinder(InventoryPlayer inventory, TileThermalBinder tileEntity) {

        addSlotToContainer(new Slot(tileEntity, TileThermalBinder.INPUT_SLOT_1, 78, 18));
        addSlotToContainer(new Slot(tileEntity, TileThermalBinder.INPUT_SLOT_2, 113, 18));
        addSlotToContainer(new Slot(tileEntity, TileThermalBinder.INPUT_SLOT_3, 78, 52));
        addSlotToContainer(new Slot(tileEntity, TileThermalBinder.INPUT_SLOT_4, 113, 52));
        addSlotToContainer(new Slot(tileEntity, TileThermalBinder.MB_SLOT, 43, 34));
        addSlotToContainer(new Slot(tileEntity, TileThermalBinder.INGOT_SLOT, 142, 10));
        bindPlayerInventory(inventory, 8, 84);
    }
}

File: src/main/scala/com/dyonovan/neotech/common/container/machines/ContainerThermalBinder.scala
package com.dyonovan.neotech.common.container.machines

import com.dyonovan.neotech.common.tiles.machines.TileThermalBinder
import com.teambr.bookshelf.common.container.BaseContainer
import net.minecraft.entity.player.InventoryPlayer

/**
 * This file was created for NeoTech
 *
 * NeoTech is licensed under the
 * Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License:
 * http://creativecommons.org/licenses/by-nc-sa/4.0/
 *
 * @author Dyonovan
 * @since August 21, 2015
 */
class ContainerThermalBinder(playerInventory: InventoryPlayer, tileEntity: TileThermalBinder) extends
        ContainerAbstractMachine(playerInventory, tileEntity) {

    addSlotToContainer(new RestrictedSlot(tileEntity, 0, 67, 22))
    addSlotToContainer(new RestrictedSlot(tileEntity, 1, 92, 22))
    addSlotToContainer(new RestrictedSlot(tileEntity, 2, 67, 67))
    addSlotToContainer(new RestrictedSlot(tileEntity, 3, 92, 67))
    addSlotToContainer(new RestrictedSlot(tileEntity, 4, 38, 44))
    addSlotToContainer(new RestrictedSlot(tileEntity, 5, 122, 44))
    addPlayerInventorySlots(8, 103)

}
